homepage.html

```diff
@@ -57,17 +57,17 @@
 		</header>
 			<section class="top-container">            
 				<div class="container-text">
-					<h1>Make your app the<br>
+					<h1 class="container-text-1">Make your app the<br>
```

Comment:
Pentru tag-urile h (ex, h1, h2, h3) de obicei este ok sa pui in denumirea clasei title este mult mai sugestiv. Se poate confunda clasa cu paragrap. Te rog pune title in loc de text.